File: Chapter01/webServer/web.js
const http = require("http");

const hostname = "127.0.0.1";
const port = 8181;

const server = http.createServer((req, res) => {
  res.setHeader("Content-Type", "text/plain");
  res.end(`Hello from JavaScript, got ${req.url}!`);
});

server.listen(port, hostname, () => {
  console.log(`Server running at http://${hostname}:${port}/`);
});

File: Chapter01/webServer/web.go
package main

import (
	"io"
	"net/http"
)

func hello(w http.ResponseWriter, req *http.Request) {
	w.Header().Add("Content-Type", "text/plain")
	io.WriteString(w, "Hello from Go!")
}

func main() {
	http.HandleFunc("/", hello)
	http.ListenAndServe(":8181", nil)
}
